File: Dev_Lab Unofficial/Python/โจทย์ FizzBuzz ในตำนาน.py
a = int(input())
if a%3 == 0 and a%2 == 0:
  print('FizzBuzz')
elif a%3 == 0 and a%2 != 0:
  print('Fizz')
elif a%3 != 0 and a%2 == 0:
  print('Buzz')

File: Dev_Lab Unofficial/C++/โจทย์ FizzBuzz ในตำนาน.cpp
#include <iostream>
using namespace std;


int main()
{
	int n;
  cin >> n;
  if(n%3 == 0 && n%2==0){
    cout << "FizzBuzz";
  }
  else if(n %2 == 0){
    cout << "Buzz";
  }
  else if(n % 3 == 0){
    cout << "Fizz";
  }
	return 0;
}